projects/kubernetes/boot.sh

```diff
@@ -1,10 +1,20 @@
-#!/bin/bash -eu
+#!/bin/sh
+
+set -e
+
 : ${KUBE_PORT_BASE:=2222}
 : ${KUBE_VCPUS:=2}
-: ${KUBE_MEM:=4096}
+: ${KUBE_MEM:=1024}
 : ${KUBE_DISK:=4G}
 : ${KUBE_NETWORKING:=default}
 : ${KUBE_RUN_ARGS:=}
+: ${KUBE_EFI:=}
+
+[ "$(uname -s)" = "Darwin" ] && KUBE_EFI=1
+
+suffix=".iso"
+[ -n "${KUBE_EFI}" ] && suffix="-efi.iso" && uefi="--uefi"
```

Comment:
The QEMU backend supports this (supposedly, I've not tried it), so maybe this could all just be unconditionally UEFI?